validation/make.py

```diff
@@ -43,6 +43,11 @@
         "command": "make.py",
         "args": ["run-fit", "all"],
     },
+    "lightcurve": {
+        "folder": "lightcurve",
+        "command": "make.py",
+        "args": ["run-analyses", "1d", "3d", "all"],
```

Comment:
I think we don't need to require the "1d" and "3d" since the option "all" already covers the use case. Also this will likely fail. Remove "1d" and "3d".